jupyter_book/config.py

```diff
@@ -344,20 +344,20 @@ def yaml_to_sphinx(yaml: dict):
     execute = yaml.get("execute")
     if execute:
         for spx_key, yml_key in [
-            ("execution_allow_errors", "allow_errors"),
-            ("execution_in_temp", "run_in_temp"),
+            ("nb_execution_allow_errors", "allow_errors"),
```

Comment:
@AakashGfude I suspect this family of configuration will be used most extensively requiring some users to update their config to use `nb_` prefixes. @chrisjsewell re: the behaviour of myst-nb/myst-parser do the old options "work" but throw warnings to update them?